modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/edit_password_policy_assignments.jsp

```diff
@@ -59,12 +61,31 @@ SearchContainer<?> searchContainer = editPasswordPolicyAssignmentsManagementTool
 	navigationItems="<%= passwordPolicyDisplayContext.getEditPasswordPolicyAssignmentsNavigationItems(editPasswordPolicyAssignmentsManagementToolbarDisplayContext.getPortletURL()) %>"
 />
 
-<clay:management-toolbar-v2
+<clay:management-toolbar
 	actionDropdownItems="<%= editPasswordPolicyAssignmentsManagementToolbarDisplayContext.getActionDropdownItems() %>"
+	additionalProps='<%=
```

Comment:
Having already the Display Context... wouldn't be better to have this additional props inside? And then access them through e`ditPasswordPolicyAssignmentsManagementToolbarDisplayContext.getAdditionalProps()`, Just wondering, I did it in my pull and would be great to have same patterns everywhere

Reply:
Sure why not, in most cases I don't bother because I've seen this pattern in portal quite a lot but I'm not against it.

Reply:
@drewbrokke whatever you think is better works for us.

Reply:
Yes, I'm also ok with your approach @drewbrokke, I was just suggesting but each team has their own preferences ;-)

Reply:
[Updated](https://github.com/drewbrokke/liferay-portal/blob/c2ae813b8ed2cab963db8265552bab1428850271/modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/edit_password_policy_assignments.jsp#L62-L77)